apps/complaints/acceptance/pages/confirmation.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
 module.exports = {
-  url: 'confirmation'
+  url: 'confirmation',
+  content: {
+    sent: 'The complaint has been sent.',
```

Comment:
No id but there is the class alert-message if you think checking for that is better
 <div class="alert-message">
        <h1>The complaint has been sent.</h1>
 </div>